src/helpers/walletConnectNetworks.js

```diff
@@ -71,11 +71,13 @@ export const androidShowNetworksActionSheet = callback => {
       title: `Available Networks`,
     },
     idx => {
-      const { value } = androidReverseNetworkWithName(
-        androidNetworkActions[idx]
-      );
-      const chainId = ethereumUtils.getChainIdFromNetwork(value);
-      callback({ chainId, network: value });
+      if (idx) {
```

Comment:
This won't let u handle the index 0 since it's falsey

Reply:
ty ser, fixed: https://cloud.skylarbarrera.com/Screen-Recording-2021-08-09-08-10-58.mp4